youtube_dl_gui/locale/it_IT/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,603 @@
+msgid "Stop"
+msgstr "Fine"
+
+#: youtube_dl_gui/mainframe.py:101
+msgid "Info"
+msgstr "Informazioni"
+
+#: youtube_dl_gui/mainframe.py:102
+msgid "Welcome"
+msgstr "Benvenuto"
+
+#: youtube_dl_gui/mainframe.py:103
+msgid "Warning"
+msgstr "Avviso"
+
+#: youtube_dl_gui/mainframe.py:105
+msgid "Add"
+msgstr "Aggiungi"
+
+#: youtube_dl_gui/mainframe.py:106
+msgid "Download list"
+msgstr "Elenco scaricamenti"
+
+#: youtube_dl_gui/mainframe.py:107 youtube_dl_gui/mainframe.py:516
+#: youtube_dl_gui/mainframe.py:534
+msgid "Delete"
+msgstr "Elimina"
+
+#: youtube_dl_gui/mainframe.py:108
+msgid "Play"
+msgstr "Riproduci"
+
+#: youtube_dl_gui/mainframe.py:109
+msgid "Up"
+msgstr "Su"
+
+#: youtube_dl_gui/mainframe.py:110
+msgid "Down"
+msgstr "Giù"
+
+#: youtube_dl_gui/mainframe.py:111
+msgid "Reload"
+msgstr "Ricarica"
+
+#: youtube_dl_gui/mainframe.py:112 youtube_dl_gui/mainframe.py:448
+#: youtube_dl_gui/mainframe.py:649
+msgid "Pause"
+msgstr "Metti in pausa"
+
+#: youtube_dl_gui/mainframe.py:113 youtube_dl_gui/mainframe.py:865
+#: youtube_dl_gui/mainframe.py:866 youtube_dl_gui/optionsframe.py:587
+msgid "Start"
+msgstr "Inizio"
+
+#: youtube_dl_gui/mainframe.py:114
+msgid "About"
+msgstr "Informazioni"
+
+#: youtube_dl_gui/mainframe.py:115
+msgid "View Log"
+msgstr "Mostra registro"
+
+#: youtube_dl_gui/mainframe.py:117
+msgid ""
+"Successfully downloaded {0} URL(s) in {1} day(s) {2} hour(s) {3} minute(s) "
+"{4} second(s)"
+msgstr ""
+"Scaricate correttamente {0} URL in {1} giorni {2} ore {3} minuti {4} secondi"
+
+#: youtube_dl_gui/mainframe.py:119
+msgid "Downloads completed"
+msgstr "Scaricamenti completati"
+
+#: youtube_dl_gui/mainframe.py:120
+msgid ""
+"Total Progress: {0:.1f}% | Queued ({1}) Paused ({2}) Active ({3}) Completed "
+"({4}) Error ({5})"
+msgstr ""
+"Progresso complessivo: {0:.1f}% | Accodati ({1}) In pausa ({2}) Attivi ({3}) "
+"Completati ({4}) Con errori ({5})"
+
+#: youtube_dl_gui/mainframe.py:121
+msgid "Stopping downloads"
+msgstr "Arresto scaricamenti"
+
+#: youtube_dl_gui/mainframe.py:122
+msgid "Downloads stopped"
+msgstr "Scaricamenti arrestati"
+
+#: youtube_dl_gui/mainframe.py:123
+msgid "You need to provide at least one URL"
+msgstr "Devi fornire almeno una URL"
+
+#: youtube_dl_gui/mainframe.py:124
+msgid "Downloads started"
+msgstr "Scaricamenti avviati"
+
+#: youtube_dl_gui/mainframe.py:125
+msgid "Choose Directory"
+msgstr "Scegli cartella"
+
+#: youtube_dl_gui/mainframe.py:127
+msgid "Download in progress. Please wait for all downloads to complete"
+msgstr ""
+"Scaricamento in corso. Attendi che tutti gli scaricamenti vengano completati"
+
+#: youtube_dl_gui/mainframe.py:128
+msgid "Update already in progress"
+msgstr "Aggiornamento già in corso"
+
+#: youtube_dl_gui/mainframe.py:130
+msgid "Downloading latest youtube-dl. Please wait..."
+msgstr "Scaricamento youtube-dl più recente. Attendere..."
+
+# Scaricamento youtube-dl non riuscito [{0}]
```

Comment:
That should probably be in the **msgstr** field below. Don't know how this ended up here.